[master] Update for cc Enforcer's dropins folder with helm artifacts (#6415)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md

docker build -t myimages/choreo-connect-router:{{choreo_connect.version}} -f Dockerfile .
```

> ### Mount Dropins Directory to a Choreo Connect Helm Deployment

Reviewer: I think we do not want to mention about helm, because this is prod guide page
```suggestion
### Mount Files into Dropins Directory (Optionally)
```

1. If you are using Choreo Connect as a Kubernetes Helm deployment, you can mount the dropins directory by following below instructions. 
   
      1. Create a configmap file to define the JAR that you are using with the Enforcer.

        ```bash
         kubectl create configmap configMapName1 --from-file=path/to/jar
        ```

      2. In the `values.yaml` file, update the `wso2.deployment.gatewayRuntime.enforcer.dropins` array with your configmap name(s).

        ```bash tab='Example'
         wso2:
            deployment:
                gatewayRuntime:
                    enforcer:
                        dropins:
                            - configMapName1
                            - configMapName2
        ```

> 2. If you are having a customised Enforcer Image (explained in the Production Deployment Guideline), follow the steps explained below considering the `values.yaml` file to mount the dropins folder. 

Reviewer: Option 2:
Instead of mounting the JAR file to dropins directory, you can burn JAR files to the Enforcer docker image. Please check <link> on how to create a custom docker image.

>       1. Define `wso2.deployment.gatewayRuntime.enforcer.dropinsMountEmptyDir` value as `false`.

Reviewer: ```suggestion
      1. Set `wso2.deployment.gatewayRuntime.enforcer.dropinsMountEmptyDir` value as `false`. Otherwise, the droppings already in the Docker image will be replaced with a Kubernetes empty dir.
```

>       2. Update `wso2.deployment.gatewayRuntime.enforcer.imageName` and `wso2.deployment.gatewayRuntime.enforcer.imageTag` values to match with your docker image.

Reviewer: ```suggestion
      2. Set `wso2.deployment.gatewayRuntime.enforcer.imageName` and `wso2.deployment.gatewayRuntime.enforcer.imageTag` values to match with your docker image.
```

docker build -t myimages/choreo-connect-router:{{choreo_connect.version}} -f Dockerfile .
```

### Mount files into the Dropins Directory (Optional)

**Option 1**

> If you are using Choreo Connect as a Kubernetes Helm deployment, you can mount the dropins directory by following below instructions. 

Reviewer 2: ```suggestion
If you are using Choreo Connect as a Kubernetes Helm deployment, you can mount the dropins directory by following the instructions below. 
```
